src/app/(app)/globals.css

```diff
@@ -2,82 +2,81 @@
 @tailwind components;
 @tailwind utilities;
 
+@import url('../../components/client/slider/productSlider.css');
```

Comment:
better add import in component where it use

Reply:
Done